cfme/common/__init__.py

```diff
@@ -120,14 +115,14 @@ def validate_properties(self):
 
         Fails if some property does not match.
         """
-        properties = self.properties_ui()
-        assert len(properties) > 0, 'No property was found in UI'
+        self.summary.reload()
+        assert len(self.summary.properties.items()) > 0, 'No property was found in UI'
         for property_tuple in self.property_tuples:
             expected_value = str(getattr(self, property_tuple[0]))
-            shown_value = str(properties[property_tuple[1]])
-            assert(expected_value == shown_value,
-                   'Property {} has wrong value, expected {} but was {}'
-                   .format(property_tuple[1], expected_value, shown_value))
+            shown_value = self.summary.properties.items()[property_tuple[1]].text_value
```

Comment:
You always use properties, so I believe it would be beneficial to pull out the `self.summary.properties.items()` before line 119 to a local variable.